e2e/storybook/src/storybook.test.ts

```diff
@@ -2,54 +2,64 @@ import {
   checkFilesExist,
   cleanupProject,
   killPorts,
-  newProject,
   runCLI,
   runCommandUntil,
   tmpProjPath,
   uniq,
-  updateJson,
   getPackageManagerCommand,
   runCommand,
+  newProject,
+  updateJson,
+  runCypressTests,
 } from '@nrwl/e2e/utils';
 import { writeFileSync } from 'fs';
 
-describe('Storybook schematics', () => {
+describe('Storybook generators and executors for monorepos', () => {
   const previousPM = process.env.SELECTED_PM;
-
-  let reactStorybookLib: string;
-  let proj: string;
-
+  const reactStorybookLib = uniq('test-ui-lib-react');
+  const angularStorybookLib = uniq('test-ui-lib');
+  let proj;
   beforeAll(() => {
     process.env.SELECTED_PM = 'yarn';
-
-    proj = newProject();
-    reactStorybookLib = uniq('test-ui-lib-react');
-
+    proj = newProject({
+      packageManager: 'yarn',
+    });
     runCLI(`generate @nrwl/react:lib ${reactStorybookLib} --no-interactive`);
     runCLI(
-      `generate @nrwl/react:storybook-configuration ${reactStorybookLib} --generateStories --no-interactive`
+      `generate @nrwl/react:storybook-configuration ${reactStorybookLib} --generateStories --no-interactive --bundler=webpack`
+    );
+
+    createTestUILib(angularStorybookLib);
```

Comment:
bring angular tests into this suite, to reuse workspace. no need to have separate e2e suite